project/Dependencies.scala

```diff
@@ -4,7 +4,8 @@ object Dependencies {
   private val circeVersion = "0.14.3"
   private val elasticMqVersion = "1.3.14"
 
-  lazy val awsUtils = "uk.gov.nationalarchives" %% "tdr-aws-utils" % "0.1.53"
+  lazy val sqsUtils = "uk.gov.nationalarchives" %% "sqs-utils" % "0.1.55"
```

Comment:
Maybe put the version in a `private val`

Reply:
Good point, I've added one in.